source/agora/test/Failures.d

```diff
@@ -30,6 +30,7 @@ unittest
 
     TestConf conf = { threshold : 100, timeout : 1_000 };
     auto network = makeTestNetwork(conf);
+    scope (exit) network.printLogs();
```

Comment:
@TrustHenry we should remove this, because the nodes will be `null` after the constructor fails, so we cannot call their `printLog` method.